File: python_ToSXs/python/05-uocsoboiso/105.py
x = 5
y = 7

result = 0
zmax = max(x, y)

for i in range(zmax, x * y + 1):
    if i % x == 0 and i % y == 0:
        result = i
        break

print(f"BCNN của {x} và {y} là {result}")

File: cpp_e9zMZ/cpp/05-uocso-boiso/105.cpp
#include <algorithm>
#include <iostream>
using namespace std;

int main() {
    int x = 5;
    int y = 20;
    int result = 0;
    int maxNumber = max(x, y);

    for (int i = maxNumber; i < x * y; i++) {
        if (i % x == 0 && i % y == 0) {
            result = i;
            break;
        }
    }

    cout << "BCNN của " << x << " và " << y << " là " << result;
    return 0;
}